django/db/migrations/operations/special.py

```diff
@@ -175,3 +178,5 @@ def database_backwards(self, app_label, schema_editor, from_state, to_state):
 
     def describe(self):
         return "Raw Python operation"
+
+    noop = lambda apps, schema_editor: None
```

Comment:
Won't that create an unbound method on PY2? IMO we should just use `def` with `@staticmethod`.